src/main/java/hotelreservation/controller/ReservationController.java

```diff
@@ -1,23 +1,71 @@
 package hotelreservation.controller;
 
 import hotelreservation.domain.HotelReservationHelper;
+import hotelreservation.domain.Status;
```

Comment:
when I run the unit tests this fails?  Are you even running the unit tests to check your work before committing?

Reply:
Looks like there might be a maven dependency missing from the pom file that didn't get committed?